cmake/analysis.cmake

```diff
@@ -18,10 +18,12 @@ else()
   message(STATUS "Target cppcheck enabled")
   add_custom_target(cppcheck
     COMMAND ${CPPCHECK_BIN} --xml --xml-version=2 ${CMAKE_SOURCE_DIR}
-      -I ${CMAKE_SOURCE_DIR}/irohad/ -I ${CMAKE_SOURCE_DIR}/iroha-cli/
```

Comment:
Why did you skip iroha-cli?

Reply:
cmake does not make "include_directories" on iroha-cli

Reply:
-I is "include folder"

Reply:
What about this?:
https://github.com/hyperledger/iroha/blob/035ee547d4819829aea8b1937386216e1de0baa5/iroha-cli/CMakeLists.txt#L51-L53

Reply:
okay, did not notice this. thanks.

Reply:
Fxd